leap/example.rb

```diff
@@ -1,25 +1,12 @@
 class Year
```

Comment:
The thing that I like about the existing solution is that the `divisible_by?` is much more readable. In the version that you added there are two numbers, and it's not clear which is the number being divided and which is the factor that is being checked. With the other one, there's no ambiguity.